.github/workflows/actions.yml

```
@@ -107,6 +107,8 @@ jobs:
           - git+https://github.com/conbench/datalogistik.git@${{ github.sha }}
     needs: build-wheel
     runs-on: ${{ matrix.os }}
+    env:
+        DATALOGISTIK_REPO: https://raw.githubusercontent.com/conbench/datalogistik/${{ env.GITHUB_REF_NAME }}/repo.json
```

forgot that you can't use `env` inside an `env` key :)
```suggestion
        DATALOGISTIK_REPO: https://raw.githubusercontent.com/conbench/datalogistik/${{ github.ref_name }}/repo.json
```